test/e2e/e2e.go

```diff
@@ -372,11 +373,60 @@ var _ = ginkgo.Describe("[efs-csi] EFS CSI", func() {
 			encryptInTransit := false
 			testEncryptInTransit(f, &encryptInTransit)
 		})
+
+		ginkgo.It("should successfully mount a single pod when ReadWriteOncePod is used statically", func() {
+			pvc, pv, err := createEFSPVCPV(f.ClientSet, f.Namespace.Name, f.Namespace.Name, "/",
+				map[string]string{}, []v1.PersistentVolumeAccessMode{v1.ReadWriteOncePod})
+			framework.ExpectNoError(err, "creating efs pvc & pv")
+			defer func() {
+				_ = f.ClientSet.CoreV1().PersistentVolumes().Delete(context.TODO(), pv.Name, metav1.DeleteOptions{})
+			}()
+			command := fmt.Sprintf("touch /mnt/volume1/%s-%s && trap exit TERM; while true; do sleep 1; done", f.Namespace.Name, time.Now().Format(time.RFC3339))
+			pod := e2epod.MakePod(f.Namespace.Name, nil, []*v1.PersistentVolumeClaim{pvc}, false, command)
+			pod.Spec.RestartPolicy = v1.RestartPolicyNever
+			pod, err = f.ClientSet.CoreV1().Pods(f.Namespace.Name).Create(context.TODO(), pod, metav1.CreateOptions{})
+			framework.ExpectNoError(err, "creating pod")
+
+			err = e2epod.WaitForPodRunningInNamespace(f.ClientSet, pod)
+			framework.ExpectNoError(err, "pod started running successfully")
+		})
+
+		ginkgo.It("should successfully mount a single pod when ReadWriteOncePod is used dynamically", func() {
+			immediateBinding := storagev1.VolumeBindingImmediate
+			sc := storageframework.GetStorageClass("efs.csi.aws.com", map[string]string{
+				"provisioningMode": "efs-ap",
+				"fileSystemId":     FileSystemId,
+				"directoryPerms":   "777",
+				"gidRangeStart":    "1000",
+				"gidRangeEnd":      "2000",
+				"basePath":         "/dynamic_provisioning",
+			}, &immediateBinding, EfsDriverNamespace)
+			_, err := f.ClientSet.StorageV1().StorageClasses().Create(context.TODO(), sc, metav1.CreateOptions{})
```

Comment:
Do we need any automatic cleanup logic for this `StorageClass` after the test completes?

Reply:
That sounds like a sensible idea, once I get someone on the repo to enable testing I can start debugging the tests as I'm not 100% sure they're right at the minute. Thank you for looking though